src/connection/RODSConn.class.php

```diff
@@ -18,7 +18,7 @@ function getRodsConn(RODSAccount $account) {
 }
 
 class RODSConn {
-    use RC_base, RC_connect;
+    //use RC_base, RC_connect; // stupid php5.6 and traits vs subclasses
```

Comment:
Are we not afraid of breaking backwards compability with too much code by doing this? We're using the RODSConnManager() in our code. But any code directly using RODSConn would break by using this?

I BTW like the splitting up of the huge RODSConn class into multiple traits. Think it's a good idea. Nice refactoring!

Reply:
yes, this is not backwards compatible when `RODSConn` is being use. `RODSConnManager` should be the API to use
the refactoring is needed because the the original simngle file was way to large to do any menaingful changes in it.
i found nothing better than `trait`, but feel free to use something better.